src/www/ui/search-helper.php

```diff
@@ -235,7 +235,7 @@ function GetResults($Item, $Filename, $Upload, $tag, $Page, $SizeMin, $SizeMax,
     }
   }
 
-  $Offset = $Page * $MaxPerPage;
+  $Offset = ($Page - 1) * $MaxPerPage;
```

Comment:
There is already `$page - 1` [here](https://github.com/Shruti3004/FOSSology-REST-API/pull/13/files#diff-ffe87cc10eca84010d4e33dd0e3f9170b0bc9bc5fe686f43bb2536e0bda67342R137), doing it for a second time will effectively be `$page - 2` isn't it?

Reply:
@GMishx No, actually i have seen the other apis which was written so for symmetry i have did the page -1 here instead in the parameter of `getResult()`

Reply:
Fixed that in the `generateResult` parameter

Reply:
Keeping -1 here, then everywhere else this function is called, `$Page + 1` will have to be done to get the same value. (Please see the places in the comment above).

Or do `$page - 1` where you need it (in `SearchController`) and do not touch anywhere else.

Reply:
Always try to minimize the code changes.

Reply:
@GMishx updated